typescript/scripts/bag2proto.ts

```diff
@@ -90,13 +104,16 @@ function rosMsgDefinitionToProto(typeName: string, msgDef: string): protobufjs.R
           qualifiers.push(field.type);
         }
       }
+      if (scalarTypes.includes(field.type)) {
+        isPacked = true;
+      }
       if (field.arrayLength != undefined) {
         lineComments.push(`length ${field.arrayLength}`);
       }
       fields.push(
-        `${qualifiers.join(" ")} ${field.name} = ${fieldNumber++};${
-          lineComments.length > 0 ? " // " + lineComments.join(", ") : ""
-        }`,
+        `${qualifiers.join(" ")} ${field.name} = ${fieldNumber++} ${
+          isPacked ? "[packed = true]" : ""
```

Comment:
I think proto3 uses packed=true by default for scalar types. Why do we need this?

Reply:
hm - I was reading docs that made it sound like I should add this.

Re-read the docs and yea we don't need packed. proto3 does it by default for scalar value fields. Will remove.